### PR TITLE
feat/config from dato

### PR DESCRIPTION
**Pull request by petergoes**
- chore: Install missing dependencies
- feat: Add scripts to fetch config files from dato
- chore: Ignore data files in public folder
- feat: get config from public data file
- feat: Add available configs dynamically from dato

### CODE REVIEW

File: config/dato/_query-api.mjs [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import axios from 'axios'
```
Comment (vvandijck) [inline config/dato/_query-api.mjs:1]: Is there a reason why this is a `.mjs` file? In any case, it's very well written 👍🏼

File: config/dato/available-configs.graphql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+query {
+  configs: allMenus(filter: {parent: {exists: false}}) {
+    name
+  }
+}
```
Comment (vvandijck) [inline config/dato/available-configs.graphql:5]: Missing end-of-file newline.

File: config/dato/available-configs.mjs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import fs from 'fs/promises'
+import slugify from '@sindresorhus/slugify'
+
+
```
Comment (vvandijck) [inline config/dato/available-configs.mjs:4]: 2 newlines can be removed.

**Review by vvandijck — APPROVED** [review]
A few small comments, overall it's ✅

File: config/dato/_query-api.mjs (hunk 1 above)
Comment (petergoes) [reply config/dato/_query-api.mjs:1]: yeah, with `.mjs` you can use `import` and top-level await

**Pull request merged**